src/Integration.Vsix/SonarLintDaemon/DaemonCommands.vsct

```diff
@@ -42,6 +42,11 @@
       <IDSymbol name="grpDaemonErrorList" value="0x100" />
       <IDSymbol name="cmdidCreateCFamilyReproducer" value="0x300" />
     </GuidSymbol>
+
+    <!-- VS Code Analysis menu/group Guid:IDs -->
+    <GuidSymbol name="guidCodeAnalysisMenuGroup" value="{FBC1A65B-A334-4EC8-8FBE-DB1C56335FB6}">
```

Comment:
This was in the PackageCommands.vsct but used also by a command in this file: https://github.com/SonarSource/sonarlint-visualstudio/blob/7f57da532ea85d27ec9072bfaa010f09fc7801cd/src/Integration.Vsix/SonarLintDaemon/DaemonCommands.vsct#L27

It's not needed there anymore so I moved it here